jina/importer.py

```diff
@@ -277,7 +277,12 @@ def _path_import(absolute_path: str) -> Optional[ModuleType]:
             spec = importlib.util.spec_from_file_location('jinahub', absolute_path)
             module = importlib.util.module_from_spec(spec)
             user_module_name = os.path.splitext(os.path.basename(absolute_path))[0]
-            sys.modules[f'{spec.name}.{user_module_name}'] = module
+            if user_module_name == '__init__':
```

Comment:
this may put some problems in many hub executors right?

Reply:
no, irrelevant, magic is in `spec.loader.exec_module(module)`